code/modules/shuttle/navigation_computer.dm

```diff
@@ -102,6 +97,14 @@
 			the_eye.placement_images[I] = list(x_off, y_off)
 
 /obj/machinery/computer/camera_advanced/shuttle_docker/give_eye_control(mob/user)
+	if((locked_traits & ZTRAIT_RESERVED) && !SSmapping.level_trait(z, ZTRAIT_RESERVED))
```

Comment:
If we have reserved level in locked traits, and level has `reserved` trait we block interraction.
```suggestion
	if((locked_traits & ZTRAIT_RESERVED) && SSmapping.level_trait(z, ZTRAIT_RESERVED))
```